Updated instructions for changing environments in FAQ (#199)
spyder-ide/spyder-docs: doc/faq.rst

.. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?

>    To work with an existing environment in Spyder, change Spyder’s default Python interpreter. To do so, in the status bar click the name of current environment and then click :guilabel:`Change default environment in Preferences`.

Reviewer: ```suggestion
   To work with an existing environment in Spyder, change Spyder’s default Python interpreter.
   To do so, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
```

Fix line breaks and a few typos

   .. image:: images/faq/faq-change-environment.png
      :alt: Change default environment in Preferences option in status bar

>    This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section. Here, check the option :guilabel:`Use the following Python interpreter`. Click the dropdown next to the box below and select your prefered environment.

Reviewer: ```suggestion
   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section.
   Here, select the option :guilabel:`Use the following Python interpreter, and use the dropdown below to select your preferred environment.
```

Fix more line breaks and more typos, and simplify the last two sentences to avoid choppiness.

.. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?

   To work with an existing environment in Spyder, you need to change Spyder’s default Python interpreter.
   To do so, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.

   .. image:: images/faq/faq-change-environment.png
      :alt: Change default environment in Preferences option in status bar

   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section.

>    Here, select the option :guilabel:`Use the following Python interpreter, and use the dropdown below to select your preferred environment.

Reviewer: ```suggestion
   Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment.
```
